ops/charm.py

```diff
@@ -1079,12 +1079,17 @@ class CollectStatusEvent(LifecycleEvent):
 
     The order of priorities is as follows, from highest to lowest:
 
-    * error
+    * error (Note: will always result in an InvalidStatusError!)
```

Comment:
I wonder if we should just remove the "error" and "unknown" lines from this bulleted list, as you'd never want to use them with collect-status / `add_status`. If that's the case, I think we should just remove the new explanatory paragraph too -- it probably just adds confusion, given you can't use those.

Reply:
Decision is to revert this change in favour of https://github.com/canonical/operator/issues/1356